worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/PlatformManager.java

```diff
@@ -112,9 +114,14 @@ public synchronized void register(Platform platform) {
         // Make sure that versions are in sync
         if (firstSeenVersion != null) {
             if (!firstSeenVersion.equals(platform.getVersion())) {
-                logger.warn("Multiple ports of WorldEdit are installed but they report different versions ({} and {}). " +
-                                "If these two versions are truly different, then you may run into unexpected crashes and errors.",
-                        new Object[]{ firstSeenVersion, platform.getVersion() });
+                logger.warn(
+                    "Multiple ports of WorldEdit are installed but they report different versions ({} and {}). "
+                        +
+                        "If these two versions are truly different, then you may run into unexpected crashes and errors.",
+                    new Object[] {
+                        firstSeenVersion,
+                        platform.getVersion()
+                    });
```

Comment:
can you pass the args as varargs instead of object[]?

Reply:
Yup. Fixed in the next commit.